### PR TITLE
Update Traditional Chinese (zh_tw.json)

### PR DESCRIPTION
**Pull request by yichifauzi**
fix so many typo still need notlin4 to proofreading
Do not merge yet! Please wait for @notlin4's review before merging.

### CODE REVIEW

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 1]
```diff
@@ -54,19 +54,19 @@
...
+	"betterstats.config.gui_smooth_scroll.tooltip": "啟用後，捲動統計面板將會使用「平滑動態效果」。如果您使用低階設備或者滑鼠滾輪無法正常運作，請停用此功能。",
 	"betterstats.config.gui_mob_follow_cursor": "在介面中生物跟隨游標",
 	"betterstats.config.trust_all_servers_bss_net": "自動切換「betterstats」網路",
-	"betterstats.config.trust_all_servers_bss_net.tooltip": "當此模組安裝在伺服器上時，它提供更多功能。「betterstats」網路協議是「betterstats」伺服器與「betterstats」用戶端通訊的方式。啟用此功能後，當適用時用戶端將自動告知伺服器已安裝了「betterstats」。停用此功能則需要每次加入伺服器時手動執行此操作。",
-	"betterstats.config.allow_stats_sharing": "允許統計數據分享",
-	"betterstats.config.allow_stats_sharing.tooltip": "允許伺服器與第三方（例如伺服器上的其他玩家）分享您的統計數據。請注意，任何運行伺服器的人都可以以一種忽略此偏好的方式修改伺服器。儘管這不應該發生，但我希望通知您這是可能的。",
+	"betterstats.config.trust_all_servers_bss_net.tooltip": "當此模組安裝在伺服器上時，它提供更多功能。「betterstats」網路協定是「betterstats」伺服器與「betterstats」用戶端通訊的方式。啟用此功能後，當適用時用戶端將自動告知伺服器已安裝了「betterstats」。停用此功能則需要每次加入伺服器時手動執行此操作。",
+	"betterstats.config.allow_stats_sharing": "允許統計資訊分享",
+	"betterstats.config.allow_stats_sharing.tooltip": "允許伺服器與第三方（例如伺服器上的其他玩家）分享您的統計資訊。請注意，任何執行伺服器的人都可以以一種忽略此偏好的方式修改伺服器。儘管這不應該發生，但我希望通知您這是可能的。",
 	"betterstats.config.register_commands": "註冊指令",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:64]:
```suggestion
	"betterstats.config.register_commands": "登錄指令",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 2]
```diff
@@ -83,13 +83,13 @@
 	"betterstats.gui.hud_screen": "統計 HUD",
 	"betterstats.gui.hud_screen.pin_stat": "釘選至 HUD",
 	"betterstats.gui.hud_screen.live_stats_toggle": "切換即時 HUD 統計更新",
-	"betterstats.gui.hud_screen.tutorial_1": "對統計數據點擊右鍵（[Shift] + 右鍵）將其新增至此處。",
-	"betterstats.gui.hud_screen.tutorial_2": "對 HUD 項目點擊右鍵以進行修改或移除。",
+	"betterstats.gui.hud_screen.tutorial_1": "對統計資訊點選右鍵（[Shift] + 右鍵）將其新增至此處。",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:86]:
```suggestion
	"betterstats.gui.hud_screen.tutorial_1": "對統計資訊按下右鍵（[Shift] + 右鍵）將其新增至這裡。",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 3]
```diff
@@ -83,13 +83,13 @@
 	"betterstats.gui.hud_screen": "統計 HUD",
 	"betterstats.gui.hud_screen.pin_stat": "釘選至 HUD",
 	"betterstats.gui.hud_screen.live_stats_toggle": "切換即時 HUD 統計更新",
-	"betterstats.gui.hud_screen.tutorial_1": "對統計數據點擊右鍵（[Shift] + 右鍵）將其新增至此處。",
-	"betterstats.gui.hud_screen.tutorial_2": "對 HUD 項目點擊右鍵以進行修改或移除。",
+	"betterstats.gui.hud_screen.tutorial_1": "對統計資訊點選右鍵（[Shift] + 右鍵）將其新增至此處。",
+	"betterstats.gui.hud_screen.tutorial_2": "對 HUD 項目點選右鍵以進行修改或移除。",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:87]:
```suggestion
	"betterstats.gui.hud_screen.tutorial_2": "在 HUD 項目上按下右鍵以進行修改或移除。",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 4]
```diff
@@ -83,13 +83,13 @@
 	"betterstats.gui.hud_screen": "統計 HUD",
 	"betterstats.gui.hud_screen.pin_stat": "釘選至 HUD",
 	"betterstats.gui.hud_screen.live_stats_toggle": "切換即時 HUD 統計更新",
-	"betterstats.gui.hud_screen.tutorial_1": "對統計數據點擊右鍵（[Shift] + 右鍵）將其新增至此處。",
-	"betterstats.gui.hud_screen.tutorial_2": "對 HUD 項目點擊右鍵以進行修改或移除。",
+	"betterstats.gui.hud_screen.tutorial_1": "對統計資訊點選右鍵（[Shift] + 右鍵）將其新增至此處。",
+	"betterstats.gui.hud_screen.tutorial_2": "對 HUD 項目點選右鍵以進行修改或移除。",
 	"betterstats.gui.hud_screen.tutorial_3": "按下 Esc 鍵以關閉此畫面。",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:88]:
```suggestion
	"betterstats.gui.hud_screen.tutorial_3": "按下 Esc 以關閉這個畫面。",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 5]
```diff
@@ -83,13 +83,13 @@
...
 	"betterstats.gui.hud_screen.pin_stat": "釘選至 HUD",
 	"betterstats.gui.hud_screen.live_stats_toggle": "切換即時 HUD 統計更新",
-	"betterstats.gui.hud_screen.tutorial_1": "對統計數據點擊右鍵（[Shift] + 右鍵）將其新增至此處。",
-	"betterstats.gui.hud_screen.tutorial_2": "對 HUD 項目點擊右鍵以進行修改或移除。",
+	"betterstats.gui.hud_screen.tutorial_1": "對統計資訊點選右鍵（[Shift] + 右鍵）將其新增至此處。",
+	"betterstats.gui.hud_screen.tutorial_2": "對 HUD 項目點選右鍵以進行修改或移除。",
 	"betterstats.gui.hud_screen.tutorial_3": "按下 Esc 鍵以關閉此畫面。",
 	
 	"betterstats.network.betterstatsnetworkhandler.toggle_tooltip": "切換「betterstats」伺服器連線",
 	"betterstats.network.betterstatsnetworkhandler.consent_warning": "重要：\n啟用此功能後，伺服器將會發現您正在使用安裝了「betterstats」的修改用戶端。為了保護您的隱私，請勿在可能不允許修改用戶端或此模組的伺服器上使用此功能。\n\n啟用此功能會新增一些額外功能，這些功能僅在模組也安裝在伺服器上時可用。",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:91]:
```suggestion
	"betterstats.network.betterstatsnetworkhandler.consent_warning": "重要事項：\n啟用這項功能後，伺服器將知道你使用的是安裝了「betterstats」的模組用戶端。為保護你的隱私，請勿在可能不允許模組用戶端或這個模組的伺服器上使用這項功能。\n\n啟用這項功能後，如果伺服器也安裝了該模組，將有額外功能可用。",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 6]
```diff
@@ -98,46 +98,46 @@
...
-	"betterstats.sas.custom": "%s 剛剛將他們的 「%s」 統計值增加到 %s。",
+	"betterstats.sas.custom": "%s 剛剛將他們的「%s」統計值增加到 %s。",
 	
   
 	"betterstats.gui.tpsbs": "第三方統計瀏覽器",
-	"betterstats.gui.tpsbs.description": "一個 GUI 介面，允許您查看來自其他來源的統計數據，例如其他玩家的統計數據。",
+	"betterstats.gui.tpsbs.description": "一個 GUI 介面，允許您查看來自其他來源的統計資訊，例如其他玩家的統計資訊。",
 	"betterstats.gui.tpsbs.tab.ssps": "同服玩家搜尋",
-	"betterstats.gui.tpsbs.tab.ssps.description": "允許您瀏覽當前連接的伺服器上存在的玩家，並從伺服器請求他們的統計數據。",
+	"betterstats.gui.tpsbs.tab.ssps.description": "允許您瀏覽當前連接的伺服器上存在的玩家，並從伺服器請求他們的統計資訊。",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:107]:
```suggestion
	"betterstats.gui.tpsbs.tab.ssps.description": "讓你瀏覽目前伺服器上的玩家，並請求伺服器提供他們的統計資訊。",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 7]
```diff
@@ -98,46 +98,46 @@
...
 	"betterstats.gui.tpsbs.tab.ssps.requirements": "此功能要求您和伺服器都安裝了最新（或更好是相同）版本的此模組。",
-	"betterstats.gui.tpsbs.tab.ssps.player_not_found": "伺服器找不到您請求的玩家名稱。\n\n請確保在重試之前符合以下條件：\n1. 確保名稱拼寫和大小寫正確\n2. 請求的玩家在線上\n3. 請求的玩家安裝了此模組\n4. 請求的玩家在其模組配置中啟用了 「允許統計數據分享」",
+	"betterstats.gui.tpsbs.tab.ssps.player_not_found": "伺服器找不到您請求的玩家名稱。\n\n請確保在重試之前符合以下條件：\n1. 確保名稱拼字和大小寫正確\n2. 請求的玩家在線上\n3. 請求的玩家安裝了此模組\n4. 請求的玩家在其模組設定中啟用了「允許統計資訊分享」",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:109]:
```suggestion
	"betterstats.gui.tpsbs.tab.ssps.player_not_found": "伺服器找不到你請求的玩家名稱。\n\n請確保滿足下列條件後再重試：\n1. 名稱拼字和大小寫正確\n2. 請求的玩家在線上\n3. 請求的玩家已安裝這個模組\n4. 請求的玩家已在這個模組的設置中啟用「統計資訊分享」",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 8]
```diff
@@ -98,46 +98,46 @@
...
 	
 	"betterstats.gui.tpsbs.tab.qs": "快速分享",
-	"betterstats.gui.tpsbs.tab.qs.description": "僅需點擊一個按鈕，即可快速與使用此模組的任何人分享您的統計數據，無論他們現在身在何處。",
-	"betterstats.gui.tpsbs.tab.qs.tos_notice": "快速分享是一個利用我（模組開發者）的伺服器運作的功能。因此，它適用於我的服務條款和隱私政策。",
-	"betterstats.gui.tpsbs.tab.qs.abuse_notice": "為防止垃圾郵件和濫用，可能會適用速率限制和其他限制。",
-	"betterstats.gui.tpsbs.tab.qs.step_1": "§e第一步:\n§7點擊按鈕快速分享您當前查看的統計文件。分享完成後，您將獲得一個 「快速分享代碼」。",
-	"betterstats.gui.tpsbs.tab.qs.step_2": "§e第二步:\n§7接收方輸入 「快速分享代碼」，並點擊按鈕下載和查看分享的統計數據。",
+	"betterstats.gui.tpsbs.tab.qs.description": "僅需點選一個按鈕，即可快速與使用此模組的任何人分享您的統計資訊，無論他們現在身在何處。",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:112]:
```suggestion
	"betterstats.gui.tpsbs.tab.qs.description": "只需點選一個按鈕，即可快速與使用這個模組的任何人分享你的統計資訊，無論他們身在何處。",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 9]
```diff
@@ -98,46 +98,46 @@
...
+	"betterstats.gui.tpsbs.tab.qs.tos_notice": "快速分享是一個利用我（模組開發者）的伺服器運作的功能。因此，它適用於我的服務條款和隱私權政策。",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:113]:
```suggestion
	"betterstats.gui.tpsbs.tab.qs.tos_notice": "快速分享功能使用我的（模組開發者的）伺服器。因此，需要遵守我的《服務條款》和《隱私權政策》。",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 10]
```diff
@@ -98,46 +98,46 @@
...
+	"betterstats.gui.tpsbs.tab.qs.abuse_notice": "為防止垃圾訊息和濫用，可能會適用速率限制和其他限制。",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:114]:
```suggestion
	"betterstats.gui.tpsbs.tab.qs.abuse_notice": "為防止垃圾訊息和濫用，可能會有速率限制以及其他限制。",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 11]
```diff
@@ -98,46 +98,46 @@
...
+	"betterstats.gui.tpsbs.tab.qs.step_1": "§e第一步：\n§7點選按鈕快速分享您當前查看的統計檔案。分享完成後，您將獲得一個「快速分享代碼」。",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:115]:
```suggestion
	"betterstats.gui.tpsbs.tab.qs.step_1": "§e第一步：\n§7點選按鈕快速分享你目前查看的統計檔案。之後，會上傳統計資訊，你會取得「快速分享碼」。",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 12]
```diff
@@ -98,46 +98,46 @@
...
+	"betterstats.gui.tpsbs.tab.qs.step_2": "§e第二步：\n§7接收方輸入「快速分享代碼」，並點選按鈕下載和查看分享的統計資訊。",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:116]:
```suggestion
	"betterstats.gui.tpsbs.tab.qs.step_2": "§e第二步：\n§7需要查看共享統計資訊的人輸入「快速分享碼」，然後點選按鈕下載並查看共用的統計資訊。",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 13]
```diff
@@ -98,46 +98,46 @@
...
 	"betterstats.gui.tpsbs.tab.qs.step_2.enter_qscode": "在此輸入快速分享代碼",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:117]:
```suggestion
	"betterstats.gui.tpsbs.tab.qs.step_2.enter_qscode": "在這裡輸入快速分享碼",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 14]
```diff
@@ -98,46 +98,46 @@
 	"betterstats.sas.first_death.hc1": "%s 剛剛死亡了，這是他們的第一次。好消息是，這可能不會再次發生。",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:98]:
```suggestion
	"betterstats.sas.first_death.hc1": "%s 剛剛第一次死亡。好消息是，這可能不會再次發生。",

```

File: betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json [hunk 15]
```diff
@@ -98,46 +98,46 @@
...
+	"betterstats.sas.custom": "%s 剛剛將他們的「%s」統計值增加到 %s。",
 	
   
 	"betterstats.gui.tpsbs": "第三方統計瀏覽器",
-	"betterstats.gui.tpsbs.description": "一個 GUI 介面，允許您查看來自其他來源的統計數據，例如其他玩家的統計數據。",
+	"betterstats.gui.tpsbs.description": "一個 GUI 介面，允許您查看來自其他來源的統計資訊，例如其他玩家的統計資訊。",
 	"betterstats.gui.tpsbs.tab.ssps": "同服玩家搜尋",
-	"betterstats.gui.tpsbs.tab.ssps.description": "允許您瀏覽當前連接的伺服器上存在的玩家，並從伺服器請求他們的統計數據。",
+	"betterstats.gui.tpsbs.tab.ssps.description": "允許您瀏覽當前連接的伺服器上存在的玩家，並從伺服器請求他們的統計資訊。",
 	"betterstats.gui.tpsbs.tab.ssps.requirements": "此功能要求您和伺服器都安裝了最新（或更好是相同）版本的此模組。",
```
Comment (notlin4) [inline betterstats-3-fabric-1.21/src/main/resources/assets/betterstats/lang/zh_tw.json:108]:
```suggestion
	"betterstats.gui.tpsbs.tab.ssps.requirements": "這項功能要求你和伺服器都安裝了這個模組的最新版本（或更好是相同版本）。",

```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Review by notlin4 — APPROVED** [review]

**Comment by TheCSDev** [comment]
Amazing. Thank you so much to both of you. Will merge now and likely publish an update later today.

**Pull request merged**